### DISCUSSION TITLE
[Feature] toggle to highlight recognized faces in a photo without opening details

### DISCUSSION THREAD

**Discussion by Andreaux** [immich-app/immich#13390] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

If you open the details pane and hover over the faces, it will show you where that particular person is in the frame, which is excellent. However, it would be nice to have a 3-state toggle somewhere without having to go and open the details pane that would 1. highlight none, 2. highlight only the person you're viewing and 3. highlight all the recognized faces in the frame.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#3489
